src/crimson/os/seastore/extent_placement_manager.cc

```diff
@@ -419,29 +419,18 @@ ExtentPlacementManager::BackgroundProcess::run_until_halt()
   );
 }
 
-ExtentPlacementManager::BackgroundProcess::reserve_result_t
+ExtentPlacementManager::BackgroundProcess::reserve_io_result_t
 ExtentPlacementManager::BackgroundProcess::try_reserve(
-    const projected_usage_t &usage)
+    const io_usage_t &usage)
 {
-  reserve_result_t res {
-    trimmer->try_reserve_inline_usage(usage.inline_usage),
-    main_cleaner->try_reserve_projected_usage(usage.inline_usage + usage.ool_usage)
```

Comment:
Looks this refactor commit introduces unexpected behavior change.

`main_cleaner` was reserving inline_usage and main_ool_usage before, but only main_ool_usage after.

This is why in https://github.com/ceph/ceph/pull/47974#discussion_r1061292389, the name is `cleaner_usage_t::main_usage`, not `main_ool_usage`.